JsonToAvro/src/main/java/cheetah/exmaple/job/JsonToAvroMapper.java

```diff
@@ -0,0 +1,19 @@
+package cheetah.exmaple.job;
```

Comment:
```suggestion
package cheetah.example.job;
```

Mappe og namespace skal lige omdøbes